src/OpenTelemetry.Exporter.Geneva/GenevaLogExporter.cs

```diff
@@ -44,6 +45,8 @@ public class GenevaLogExporter : GenevaBaseExporter<LogRecord>
     };
 
     private readonly IDataTransport m_dataTransport;
+    private static readonly ConcurrentDictionary<string, string> rawNameToSanitizedName = new();
```

Comment:
My major concern with ConcurrentDictionary for logs: https://devblogs.microsoft.com/pfxteam/faq-are-all-of-the-new-concurrent-collections-lock-free/

Reply:
I've added stress tests to compare against implementation without ConcurrentDictionary and with ConcurrentDictionary.
It seems the implementation with ConcurrentDictionary performs better.

Stress test results based on this branch (w/o ConcurrentDictionary):
* Total Loops: 92,801,112
* Average Loops/Second: 3,760,327
* Average CPU Cycles/Loop: 9,450

Stress test results based on this [branch](https://github.com/Yun-Ting/opentelemetry-dotnet-contrib/pull/1) (w/ ConcurrentDictionary):
* Total Loops: 121,590,331
* Average Loops/Second: 3,988,268
* Average CPU Cycles/Loop: 9,181

Reply:
I've added the baseline for comparison (current main branch) and also increase the stress test running duration to be 5 minutes.

The latest results are as follows:
baseline (main branch)
* Total Loops: 1,106,043,741
* Average Loops/Second: 3,057,844
* Average CPU Cycles/Loop: 6,867

No Cache with passThruTableMappings Enabled
* Total Loops: 878,389,049
* Average Loops/Second: 2,912,971
* Average CPU Cycles/Loop: 7,193

With Cache with passThruTableMappings Enabled
* Total Loops: 889,221,235
* Average Loops/Second: 2,904,167
* Average CPU Cycles/Loop: 7,196

There is no significant perf improvement with caching. So I think ConcurrentDictionary is not a must here.